src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -626,26 +627,48 @@ export class CategoryFacet extends Component implements IAutoLayoutAdjustableIns
     return Utils.arrayEqual(this.activePath, this.options.basePath);
   }
 
+  private buildClearIcon() {
+    const title = l('Clear', this.options.title);
+    const clearIcon = $$('div', { className: 'coveo-category-facet-header-eraser coveo-facet-header-eraser' }, SVGIcons.icons.mainClear);
+
+    SVGDom.addClassToSVGInContainer(clearIcon.el, 'coveo-facet-header-eraser-svg');
+
+    const onClearClick = () => {
+      this.logAnalyticsEvent(analyticsActionCauseList.categoryFacetClear);
+      this.reset();
+    };
+
+    new AccessibleButton()
+      .withElement(clearIcon.el)
+      .withTitle(title)
```

Comment:
Is adding a title attribute that is the same as the label attribute needed? In my experience, the screen reader ends up reading e.g. "clear button clear". I find "clear button" much nicer.